CHANGES.rst

```diff
@@ -3376,7 +3376,7 @@ API Changes
     configuration items have moved, and some have been changed to science state
     values.  The old locations should continue to work until astropy 0.5, but
     deprecation warnings will be displayed.  See the `Configuration transition
-    <http://astropy.readthedocs.org/en/v0.4/config/config_0_4_transition.html>`_
+    <http://astropy.readthedocs.io/en/v0.4/config/config_0_4_transition.html>`_
```

Comment:
I think it's fine to fix this link (and ok to change it to ``data.astropy.org``) - or alternatively we just remove the link altogether and just say it's explained in the config docs.